plugins/Referrers/API.php

```diff
@@ -130,7 +130,6 @@ public function getAll($idSite, $period, $date, $segment = false)
         }
 
         $dataTable = $dataTable->mergeSubtables($labelColumn = 'referer_type', $useMetadataColumn = true);
-        $dataTable->filter('Sort', array(Metrics::INDEX_NB_VISITS, 'desc'));
```

Comment:
No longer needed since we do now always sort in GenericFilter. So we do no longer sort the entire table twice